test/mixer_test.py

```
@@ -1330,6 +1330,46 @@ def __init__(self):
+            "house_lo.flac",
+            "house_lo.opus",
+            "surfonasinewave.xm",
+        ]
+        if pygame.mixer.get_sdl_mixer_version() >= (2, 6, 0):
+            filenames.append("house_lo.mp3")
+
+        for f in filenames:
+            filename = example_path(os.path.join("data", f))
+            try:
+                sound = mixer.Sound(file=filename)
+            except pygame.error as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)